translations/pt-br/README.md

```diff
@@ -0,0 +1,175 @@
+Artigos que podem ajudar pessoas a falar em público
```

Comment:
Suggestion: Put a header here. Then it'll automatically form an anchor and I'll update the main README with a link directly to this section for PT-BR speakers.